include/setjmp.h

```diff
@@ -10,9 +10,9 @@
 #ifndef __SETJMP_H
 #define __SETJMP_H
 
-typedef int jmp_buf[10];
+typedef int jmp_buf[20];
```

Comment:
I think it would be worthwhile to document (in a comment) exactly what's in a jmp_buf. The choice of 20 seems kind of random, and "it just works" doesn't seem enough justification. Also, why int? Why not uintptr?